episode-10-switching-aws-to-roar/config/initializers/mime_types.rb

```diff
@@ -2,3 +2,11 @@
 
 # Add new mime types for use in respond_to blocks:
 # Mime::Type.register "text/richtext", :rtf
+
+# Do I need this?
```

Comment:
I tried this, maybe wondering JSON-API could be enabled this way, and it seemed to get a bit further, but I'm getting this instead:

```
     NoMethodError:
       undefined method `to_json_api' for #<#<Class:0x007fecdbcf67d0>:0x007fecdbb30220>
       Did you mean?  to_json
     # ./app/controllers/products_controller.rb:12:in `index'
     # ./spec/support/default_format.rb:12:in `block in <module:RequestHelpersCustomized>'
     # ./spec/requests/products_controller_spec.rb:35:in `block (4 levels) in <top (required)>'
     # ------------------
     # --- Caused by: ---
     # ActionView::MissingTemplate:
     #   Missing template products/index, application/index with {:locale=>[:en], :formats=>[:json_api], :variants=>[], :handlers=>[:raw, :erb, :html, :builder, :ruby]}. Searched in:
     #     * "/Users/psturgeon/src/livecoding-apisyouwonthate/episode-10-switching-aws-to-roar/app/views"
     #   ./app/controllers/products_controller.rb:12:in `index'

Finished in 0.28862 seconds (files took 1.1 seconds to load)
```